public/content/documentation/web/page-level/main-landmark.md

```diff
@@ -0,0 +1,124 @@
+## General Notes
+
+How to test a main landmark
+
+## Condensed
+
+### #a11y - Web Accessibility Acceptance Criteria
+
+How to test a main landmark
+
+1. Test keyboard only, then screen reader + keyboard actions
+
+   - Skip-links: Focus moves directly to the main content area
+   - Tab: Links and buttons within the main area are focusable
+
+2. Test mobile screenreader gestures
+
+   - Swipe: Focus moves within main area
+
+3. Listen to screenreader output on all devices
+
+   - Role: It is discoverable with screenreader shortcuts as main landmark
+   - Group: It contains the content portion of the page starting with the H1
+
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/page-level/main-landmark](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/page-level/main-landmark)
+
+## Gherkin
+
+### #a11y - Web Accessibility Acceptance Criteria
+
+How to test a main landmark
+
+GIVEN THAT I am on a page with a main landmark
+
+1. Keyboard for mobile & desktop
+
+   - WHEN I use the arrow keys to browse the main content I SEE the screen scroll through the page
+
+2. Desktop screenreader
+
+   - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND 
+   - I use the arrow keys to browse the main content
+      - I HEAR It is discoverable with screenreader shortcuts as main landmark
+      - I HEAR It contains the content portion of the page starting with the H1
+
+
+3. Mobile screenreader
+
+   - WHEN I use a mobile screenreader (Talkback, VoiceOver) AND
+   - I swipe to elements in the header
+      - I HEAR It is discoverable with screenreader shortcuts as main landmark
+      - I HEAR It contains the content portion of the page starting with the H1
+
+
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/page-level/main-landmark](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/page-level/main-landmark)
+
+## Code examples
+
+### Use semantic HTML
+
+This semantic HTML contains all accessibility features by default.
+
+```html
+<a href="#example-main">Skip to main content</a>
+<a href="#">Not main content</a>
+<main tabindex="-1" id="example-main">
+  <h1>About main content</h1>
+  <p>The main content of the page belongs here.</p>
+  <p><a href="#">Focus moves here next</a></p>
+</main>
+```
+<example>
+   <a href="#example-main">Skip to main content</a>
```

Comment:
this is inheriting weird styles from the regular main element. let's pair on this